db/migrate/20210119101158_create_payment_intents.rb

```diff
@@ -0,0 +1,16 @@
+class CreatePaymentIntents < ActiveRecord::Migration[5.0]
+  disable_ddl_transaction!
+
+  def change
+    create_table :payment_intents do |t|
```

Comment:
We should consider if we also want to add the column `tenant_id` and their corresponding triggers or no need 😄